client/components/date-picker/nav-bar.jsx

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import { noop } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+const handleMonthClick = ( onClick = noop ) => event => {
+	event.preventDefault();
+	onClick();
+};
+
+export const DatePickerNavBar = ( {
+	nextMonth,
+	previousMonth,
+	onPreviousClick,
+	onNextClick,
+	className,
+	localeUtils,
+} ) => {
+	const classes = classNames( 'date-picker__nav-bar', {
+		[ className ]: !! className,
+	} );
+	return (
+		<div className={ classes }>
+			<button
+				className="date-picker__previous-month button"
+				type="button"
+				aria-label={ translate( 'Previous month ', {
+					comment: 'Aria label for date picker controls',
+				} ) }
+				onClick={ handleMonthClick( onPreviousClick ) }
+			>
+				{ localeUtils.formatMonthShort( previousMonth ) }
+			</button>
+			<button
+				className="date-picker__next-month button"
+				type="button"
+				aria-label={ translate( 'Next month ', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Next Month' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).